EncounterMe/EncounterMe/Views/MainPage.xaml.cs

```diff
@@ -37,11 +37,47 @@ protected override void OnAppearing()
 
             AllUsers = App.s_mapPinService.AllUsers;
             AllUsers.Sort((a, b) => b.Score.CompareTo(a.Score));
+            
 
             if (AllUsers.Count > 5)
+            {
                 list.ItemsSource = AllUsers.Take(5);
+
+                var userPlace = AllUsers.FindIndex(u => u.Id == User.Id);
+                Console.WriteLine("Userio indeksas " + userPlace);
+
+                if(userPlace == 5)
```

Comment:
O userPlace skaičiuojasi nuo 0, nes jei ne, tai juk tai take'ini 5, tai penktas ir bus tas paskutinis sąraše ar ne?

Reply:
Skaičiuojasi nuo 0, nes imamas kaip indeksas to list'o